### PR TITLE
IoApic convenience function to get a Range of IVs

### PR DESCRIPTION
**Pull request by nolandda**
Added the convenance func requested by @dlrobertson in #88 and added it to ioapic_for_gsi as suggested in the issue.

### CODE REVIEW

File: mythril/src/ioapic.rs [hunk 1]
```diff
@@ -49,11 +50,9 @@ static IOAPICS: RoAfterInit<ArrayVec<[IoApic; MAX_IOAPIC_COUNT]>> =
 // TODO(alschwalm): Support InterruptSourceOverride
 fn ioapic_for_gsi(gsi: u32) -> Option<(&'static IoApic, u8)> {
     for ioapic in IOAPICS.iter() {
-        let entries = ioapic.max_redirection_entry() as u32;
-        let base = ioapic.gsi_base;
-        if gsi > base && gsi < base + entries {
-            return Some((ioapic, (gsi - base) as u8));
-        }
+	if ioapic.get_ivec_range().contains(&gsi) {
```
Comment (dlrobertson) [inline mythril/src/ioapic.rs:53]: nit: 4 spaces :smile:

File: mythril/src/ioapic.rs [hunk 2]
```diff
@@ -285,6 +284,13 @@ impl IoApic {
             Ok(())
         }
     }
+
+    /// convenience function to get a Range of the interrupt vectors
+    /// that should be associated with this IoApic.
+    pub fn get_ivec_range(&self) -> Range<u32> {
+	return Range{ start: self.gsi_base, end: self.gsi_base +
```
Comment (dlrobertson) [inline mythril/src/ioapic.rs:291]: nit: 4 spaces :smile:

**Review by dlrobertson — COMMENTED** [review]
Thanks for the PR! Just some small style nits.

File: mythril/src/ioapic.rs (hunk 1 above)
Comment (nolandda) [reply mythril/src/ioapic.rs]: ✅

File: mythril/src/ioapic.rs (hunk 2 above)
Comment (nolandda) [reply mythril/src/ioapic.rs]: ✅

**Comment by dlrobertson** [comment]
Thanks!

**Pull request merged**